mlos_bench/mlos_bench/config/environments/apps/fio/fio.jsonc

```diff
@@ -0,0 +1,72 @@
+{
+    "name": "FIO Composite Env",
+    "class": "mlos_bench.environments.composite_env.CompositeEnv",
+
+    "config": {
+        "children": [
+            {
+                "name": "Upload FIO job file",
+                "class": "mlos_bench.environments.local.LocalFileShareEnv",
+                "config": {
+                    "required_args": [
+                        "experiment_id",
+                        "trial_id"
+                    ],
+                    "upload": [
+                        {
+                            "from": "environments/apps/fio/scripts/remote/",
+                            "to": "$experiment_id/$trial_id/scripts"
+                        }
+                    ]
+                }
+            },
+            {
+                "name": "Run FIO remotely",
+                "class": "mlos_bench.environments.remote.RemoteEnv",
+                "config": {
+                    "required_args": [
+                        "vmName",
+                        "experiment_id",
+                        "trial_id",
+                        "mountPoint"
+                    ],
+                    "const_args": {
+                        "fio_target": "/dev/sda"
```

Comment:
Did you test this?
I'm pretty sure that will break whatever filesystem is on the disk.
Probably better to use an actual file on top of a filesystem.
Without that I think this is just going to directly write to the block device that the OS is on and seriously corrupt the filesystem.

Reply:
Indeed this is a problem.  I know this because I tried it on a VM of mine and had to reinstall it 😢

Reply:
oops, thank you for double checking it. I was lucky to not experience this issue on my VM. I was under the impression that it reads and writes the same data (after all, `/dev/sda` comes from the official example). Anyway, I'll fix it in a second

Reply:
done

Reply:
Yeah, fio is often used to test raw block device performance, so that example makes sense.
Whether or not you ran into it is probably a matter of how long the VM was up and whether or not you accessed the areas of the FS in a non-cached fashion.  I noticed it in `dmesg` output.